frontend/components/product-list/sections/FilterableProductsSection/FacetsAccordion.tsx

```diff
@@ -98,7 +98,24 @@ export const FacetAccordionItem = forwardRef<FacetAccordionItemProps, 'div'>(
       const isDisabled = isProductListEmpty || !hasApplicableRefinements;
 
       const formattedFacetName = formatFacetName(attribute);
-      const isHidden = !hasApplicableRefinements && !isProductListEmpty;
+      let isHidden = !hasApplicableRefinements && !isProductListEmpty;
+      const emptyProductList = isProductListEmpty && !hasApplicableRefinements;
```

Comment:
Naming: At first glance, it's unclear what the difference between `isProductListEmpty` and `emptyProductList` is. Maybe something like `isUnrefinedEmptyProductList`? That is more verbose tho

Reply:
Resolved 451fdd85a5550cd34a1f611c1383149be57d57ac